amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/job/DispositionLifecycleJobExecuter.java

```diff
@@ -300,6 +301,10 @@ private void executeAction(final List<NodeRef> actionNodes)
 
                 if (isFrozenOrHasFrozenChildren(parent.getParentRef()))
                 {
+                    if (logger.isDebugEnabled()) {
```

Comment:
Please can you change the logger implementation to use slf4j. By doing this we can remove the need for the if statement.

Reply:
Having seen the extent of the PR maybe changing the logger is best done in a separate PR.

Reply:
yes, We will change Logger Implementation of slf4j in separate PR